src/ts/UIConfig.ts

```diff
@@ -169,4 +169,41 @@ export interface UIConfig {
    * Default: false
    */
   includeWatermark?: boolean;
+  /**
+   * Configure Shadow DOM rendering.
+   * Enable it with:
+   * `shadowDomConfig: { enabled: true }`
```

Comment:
If you change the shorthand to be supported, let's show both here?


```suggestion
   * `shadowDom: true`
   * `shadowDom: { enabled: true }` (especially if you need to set more configuration options)
```

Reply:
[90750c1](https://github.com/bitmovin/bitmovin-player-ui/pull/808/commits/90750c15b99a52cbf477c6e5f37ba21fa9834804)